android-components/components/support/utils/src/main/java/mozilla/components/support/utils/Browsers.kt

```diff
@@ -194,6 +194,13 @@ class Browsers private constructor(
         return browsers.containsKey(browser.packageName)
     }
 
+    /**
+     * Does this device have browser with [packageName] installed?
```

Comment:
should we have tests for this in BrowsersTest similar to the `isInstalled` func above?

Reply:
Added tests in `BrowsersTest`.  Thanks!